robocop/checkers/duplications.py

```diff
@@ -146,18 +153,54 @@ class SectionHeadersChecker(VisitorChecker):
     }
 
     def __init__(self):
-        self.sections_order = {
-            Token.SETTING_HEADER: 0,
-            Token.VARIABLE_HEADER: 1,
-            Token.TESTCASE_HEADER: 2,
-            'TASK HEADER': 2,
-            Token.KEYWORD_HEADER: 4
-        }
+        self.sections_order = {}
+        self.section_order_str = None
+        self.configure('sections_order', 'settings,variables,testcases,keywords')
         self.sections_by_order = []
         self.sections_by_existence = set()
         super().__init__()
 
-    def visit_File(self, node):
+    def configure(self, param, value):
+        section_map = {
+            'settings': Token.SETTING_HEADER,
+            'variables': Token.VARIABLE_HEADER,
```

Comment:
@bhirsz  Why no option with singular 'variable' and 'setting'?

Reply:
I guess it doesn't hurt to add it if I covered other cases - I will do it in separate PR